### PR TITLE
Automatic table of content generation in tutorials

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

I found how to automatically generate the Table of Content based on the section title using Jekyll
I added this functionality and change the tutorials so they can use it.
It will make easier the writing and maintenance of tutorial: no need to check the links and the title...

So information, I excluded the Introduction and Conclusion sections of the TOC.

Bérénice

### CODE REVIEW

File: topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
----
+topics/proteomics/tutorials/database-handling/tutorial.md---
```
Comment (shiltemann) [inline topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.md]: think this change was unintentional?

**Comment by shiltemann** [comment]
neat! :)

File: topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.md (hunk 1 above)
Comment (bebatut) [inline topics/metagenomics/tutorials/mothur-miseq-sop/tutorial.md]: ooops

**Comment by bebatut** [comment]
Anyone for merging this one?

**Pull request merged**